ui/media/js/dnd.js

```diff
@@ -306,19 +313,25 @@ function restoreTaskToUI(task, fieldsToSkip) {
     }
     if (!('mask' in task.reqBody)) {
         maskSetting.checked = false
+        maskSetting.dispatchEvent(new Event("click"))
```

Comment:
Shouldn't this "click" event only be dispatched if maskSetting.checked was true? Otherwise this isn't a state change.

Reply:
Fixed.